pulsar-websocket/src/main/java/org/apache/pulsar/websocket/WebSocketProducerServlet.java

```diff
@@ -35,6 +35,6 @@ public WebSocketProducerServlet(WebSocketService service) {
     @Override
     public void configure(WebSocketServletFactory factory) {
         factory.getPolicy().setMaxTextMessageSize(WebSocketService.MaxTextFrameSize);
-        factory.setCreator((req, resp) -> new ProducerHandler(service, req.getHttpServletRequest()));
+        factory.setCreator((req, resp) -> new ProducerHandler(service, req.getHttpServletRequest(), resp));
```

Comment:
> (req, resp)

Why are these variable names different from WebSocketConsumerServlet and WebSocketReaderServlet?

Reply:
There was no reason. fixed